catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,221 +13,217 @@
+        <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.AssociationsAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.experimental.ExtractedAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.ValidationAttributes"/>
+      </list>
+    </argument>
+  </bean>
+
+  <bean id="fallbackPowerpointMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+    <argument value="fallback.powerpoint"/>
+    <argument>
+      <list>
+        <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.AssociationsAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.DateTimeAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.experimental.ExtractedAttributes"/>
+        <bean class="ddf.catalog.data.impl.types.ValidationAttributes"/>
+      </list>
+    </argument>
+  </bean>
+
+  <service ref="commonTikaMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.common"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackExcelMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.excel"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackJpegMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.jpeg"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackMp4MetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.mp4"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackMpegMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.mpeg"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackOfficeDocMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.doc"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackPdfMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.pdf"/>
+    </service-properties>
+  </service>
+
+  <service ref="fallbackPowerpointMetacardType" interface="ddf.catalog.data.MetacardType">
+    <service-properties>
+      <entry key="name" value="fallback.powerpoint"/>
+    </service-properties>
+  </service>
+</blueprint>
```

Comment:
✏️This formatting of this file does look changed to me, even when I pull down your PR.